test/migratus/test/utils.clj

```diff
@@ -9,4 +9,10 @@
   (is (= {:password "1<censored>" :user "user"}
          (censor-password {:password "1234" :user "user"})))
   (is (= "uri-censored"
-         (censor-password "jdbc:postgresql://fake.rds.amazonaws.com/capital_thing?user=capital_db&password=thisIsNot123ARealPass"))))
+         (censor-password
+           "jdbc:postgresql://fake.example.org/my_dev?user=my_user&password=thisIsNot123ARealPass")))
+  (is (= {:connection-uri "uri-censored"}
+         (censor-password {:connection-uri "jdbc:postgresql://fake.example.org/my_dev?user=my_user&password=thisIsNot123ARealPass"})))
```

Comment:
Unfortunately, my previous change did not handle the case of a map with key `:connection-uri`, which is what occurs when migratus is called from luminus-migrations.